### PR TITLE
1 4 x

### PR DESCRIPTION
**Pull request by anguram-ks**
#### Checklist

- [ ] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [ ] relevant documentation is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [ ] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes:

### CODE REVIEW

**Comment by iCG-SE** [comment]
https://github.com/electron/electron/tree/v13.0.0-beta.27

**Review by iCG-SE — COMMENTED** [review]
Good?

File: .gitignore [hunk 1]
```diff
@@ -26,5 +26,4 @@ node_modules/
 *.vcxproj.user
 *.vcxproj.filters
 *.sln
-debug.log
-npm-debug.log
```
Comment (iCG-SE) [inline .gitignore:30]: Explain this line please

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,2 +1,23 @@
+<!--
+Thanks for opening an issue! A few things to keep in mind:
+
+- The issue tracker is only for bugs and feature requests.
+- Before reporting a bug, please try reproducing your issue against
+  the latest version of Electron.
+- If you need general advice, join our Slack: http://atom-slack.herokuapp.com
+-->
+
 * Electron version:
```
Comment (iCG-SE) [inline ISSUE_TEMPLATE.md:10]: V13.0.0 beta 2.7

**Comment by Syitanazeera** [comment]
> #### Description of Change
> #### Checklist
> * [x]  PR description included and stakeholders cc'd
> * [ ]  `npm test` passes
> * [ ]  tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
> * [ ]  relevant documentation is changed or added
> * [ ]  PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
> * [ ]  [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).
> 
> #### Release Notes
> Notes:

**Pull request closed without merging**